.github/workflows/CI.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests on node
+
+name: CI
+
+on:
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  validation:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16.x]
```

Comment:
Do you want to keep the matrix if we only check against node 16.x ?

Reply:
Indeed can be removed !